src/main/java/com/patsi/utils/ListHelper.java

```diff
@@ -1,10 +1,24 @@
 package com.patsi.utils;
 
 import java.util.ArrayList;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class ListHelper {
     public static <T> List<T> newList() {
         return new ArrayList<T>();
     }
+    public static <T> List<T> newListWithParam(T... elements) {
```

Comment:
Why need this helper? Why don't simply use List.of(...elements)?

Reply:
oh yes I fixed it now :D